getting-started.md

```diff
@@ -12,41 +12,64 @@
   * DNS Domain - The DNS domain of your network
+  * NTP Server - NTP Server(s) for proper time synchronization. If you have multiple DNS Servers, input needs to be space separated.
 
-*Credentials*
+*Proxy Settings (Optional)*
+  * HTTP Proxy Server - HTTP Proxy Server followed by the port and without typing http:// before (e.g. proxy.provider.com:3128)
+  * HTTPS Proxy - HTTPS Proxy Server followed by the port and without typing https:// before (e.g. proxy.provider.com:3128)
+  * Proxy Username - Optional Username for Proxy Server
+  * Proxy Password - Optional Password for Proxy Server
+  * No Proxy - Exclude internal domain suffix. Comma separated (localhost, 127.0.0.1, domain.local)
 
+*OS Credentials* (**Required**)
   * Root Password - This is the OS root password for the vCenter Event Broker Appliance
-  * OpenFaaS Password - This is the Admin password for OpenFaaS UI
 
-*vSphere*
+
+*vSphere* (**Required**)
 
   * vCenter Server - This FQDN or IP Address of your vCenter Server that you wish to associate this vCenter Event Broker Appliance to for Event subscription
   * vCenter Username - The username to login to vCenter Server, as mentioned earlier, readOnly account is sufficient
   * vCenter Password - The password to the vCenter Username
   * Disable vCenter Server TLS Verification - If you have a self-signed SSL Certificate, you will need to check this box
 
-*zDebug*
+*Event Processor Configuration* (**Required**)
+  * Event Processor - Choose either OpenFaaS (default) or AWS EventBridge and only fill in the configuration for the selected event processor
+
+*OpenFaaS Configuration* (**Required if selected as Event Processor**)
+  * Password - Password to login into OpenFaaS. Please use a secure password
+  * Advanced Settings - N/A, future use
+
+*AWS EventBridge Configuration* (**Required if selected as Event Processor**)
+  * Access Key - A valid AWS Access Key to AWS EventBridge
+  * Access Secret - A valid AWS Access Secret to AWS EventBridge
+  * Event Bus Name - Name of the AWS Event Bus to use
+  * Region - Region where Event Bus is running (e.g. us-west-2)
+  * Rule ARN - ID of the Rule ARN created in AWS EventBridge
+  * Advanced Settings - N/A, future use
 
+*zAdvanced (Optional)*
   * Debugging - When enabled, this will output a more verbose log file that can be used to troubleshoot failed deployments
+  * POD CIDR Network - Customize POD CIDR Network (Default 10.99.0.0/20)
 
 **Step 3** - Power On the vCenter Event Broker Appliance after successful deployment. Depending on your external network connectivity, it can take a few minutes while the system is being setup. You can open the VM Console to view the progress. Once everything is completed, you should see an updated login banner for the various endpoints:
 
 ```
 Appliance Status: https://[hostname]/status
 Install Logs: https://[hostname]/bootstrap
+Appliance Statistics: https://[hostname]/stats
 OpenFaaS UI: https://[hostname]
```

Comment:
Add a note that this endpoint won't be available when using EventBridge as a processor?

Reply:
Noted. However, the code is smart enough to not actually list that as an endpoint if you're using AWS EB :)